code/parse/sexp.cpp

```diff
@@ -13320,6 +13321,37 @@ void sexp_player_use_ai(int flag)
 	}
 }
 
+void sexp_set_friendly_damage_caps(int n) {
+	bool is_nan;
+	bool is_nan_forever;
+	ai_profile_t& aip = *The_mission.ai_profile;
+
+	float beam_friendly_cap = i2fl(eval_num(n, is_nan, is_nan_forever));
```

Comment:
It's a good idea to check for `is_nan` or `is_nan_forever` being set to true.  Same with the other two caps.